bedrock/firefox/templates/firefox/australis/tour-base.html

```diff
@@ -23,8 +23,6 @@
 
 {% block body_class %}sky{% endblock %}
 
-{% block site_css %}{% endblock %}
```

Comment:
Added tabzilla and sandstone to firefox less bundle which should be inherited by all the australis templates.